### PR TITLE
Feature/panda-3264

### PR DESCRIPTION
**Pull request by Qingyu-Yang**
Implemented consignment tracking feature

### CODE REVIEW

File: projects/core/src/occ/adapters/user/occ-user-order.adapter.ts [hunk 1]
```diff
@@ -28,6 +30,15 @@ export class OccUserOrderAdapter implements UserOrderAdapter {
     return this.occEndpoints.getEndpoint(orderEndpoint);
   }
 
+  protected getConsignmentTrackingEndpoint(
+    orderCode: string,
+    consignmentCode: string
+  ): string {
+    const endpoint =
+      '/orders/' + orderCode + '/consignments/' + consignmentCode + '/tracking';
```
Comment (Jerify02) [inline projects/core/src/occ/adapters/user/occ-user-order.adapter.ts]: const endpoint = \`/orders/${orderCode}/consignments/${consignmentCode}/tracking\`

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts [hunk 2]
```diff
@@ -53,11 +59,23 @@ const moduleComponents = [
...
+    ...moduleComponents,
+    TrackingEventsComponent,
+    ConsignmentTrackingComponent,
+  ],
   exports: [...moduleComponents],
-  entryComponents: [...moduleComponents],
+  entryComponents: [
+    ...moduleComponents,
+    TrackingEventsComponent,
+    ConsignmentTrackingComponent,
```
Comment (Jerify02) [inline projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts]: It is better to move `TrackingEventsComponent`&`ConsignmentTrackingComponent` into `moduleComponents`.

File: projects/core/src/occ/adapters/user/occ-user-order.adapter.ts (hunk 1 above)
Comment (Qingyu-Yang) [reply projects/core/src/occ/adapters/user/occ-user-order.adapter.ts]: fixed

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts (hunk 2 above)
Comment (Qingyu-Yang) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts]: fixed

File: projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts [hunk 3]
```diff
@@ -53,10 +61,18 @@ const moduleComponents = [
...
+      },
     }),
+    SpinnerModule,
   ],
   providers: [OrderDetailsService],
-  declarations: [...moduleComponents],
+  declarations: [
+    ...moduleComponents,
+    TrackingEventsComponent,
+    ConsignmentTrackingComponent,
```
Comment (Jerify02) [inline projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts]: Needs fix here as well.

Comment (Qingyu-Yang) [reply projects/storefrontlib/src/cms-components/myaccount/order/order-details/order-details.module.ts]: fixed

**Pull request merged**